llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -68,6 +61,21 @@
 #define YK_CONTROL_POINT_ARG_VARS_IDX 2
 #define YK_CONTROL_POINT_NUM_ARGS 3
 
+// Stackmap ID zero is reserved for the control point.
+//
+// This will need to change when we support >1 control point.
+const unsigned CPStackMapID = 0;
+
+// The number of shadow bytes required for the control point's patchpoint.
+//
+// This must be large enough to accommodate the call to patchpoint target
+// function. LLVM will assert this in debug builds.
+#if defined(__x86_64__) || defined(_M_X64)
+const unsigned CPShadow = 13;
```

Comment:
This can't be calculated in some way? If it can be asserted, I assume(d) it can be calculated...

Reply:
Good question. We just looked at the assembler and then adjusted the value until the call fit into the shadow area. Could there be calls that need more space than this? I don't know.

Reply:
OK, I get it. Let's add some of this extra justification into the comment?